site.local.yml

```diff
@@ -5,7 +5,9 @@ site:
 content:
   sources:
   - url: .
-    branches: HEAD
```

Comment:
site.local.yml have been dropped from master branch.

Reply:
Thanks for reminding me of that. I've just removed that file from the PR.